Sources/Overload/OvEditor/src/OvEditor/Panels/Console.cpp

```diff
@@ -13,6 +13,7 @@
 #include <OvUI/Widgets/Selection/CheckBox.h>
 #include <OvUI/Widgets/Visual/Separator.h>
 #include <OvUI/Widgets/Layout/Spacing.h>
+#include "OvEditor/Settings/EditorSettings.h"
```

Comment:
I know I still haven't updated to coding conventions, but moving forward, includes will be sorted in alphabetical order (lower case first) and use <> instead of ""